lib/atom-scalariform.js

```diff
@@ -5,6 +5,19 @@ var packagePath = function() {
   return atom.packages.getLoadedPackage('atom-scalariform').path;
 }
 
+var projectPath = function(fileToFormatPath) {
+  var ref = atom.project.getDirectories();
+  var projectPath = '';
+  for (var i = 0, len = ref.length; i < len; i++) {
+    var directory = ref[i];
+    if (fileToFormatPath.indexOf(directory.path) > -1) {
```

Comment:
Just reading your comments again on the PR I think I may have misunderstood the purpose of this PR, the check for === 0 should be fine.